src/Utilities/FlowAnalysis/FlowAnalysis/Framework/DataFlow/DictionaryAnalysisData.cs

```diff
@@ -56,15 +56,10 @@ public ICollection<TKey> Keys
             }
         }
 
-        public ICollection<TValue> Values
-        {
-            get
-            {
-                // "Values" might be accessed during dispose.
-                //Debug.Assert(!IsDisposed);
-                return _coreAnalysisData.Values;
-            }
-        }
+        public ICollection<TValue> Values =>
+            // "Values" might be accessed during dispose.
```

Comment:
I think we may want to consider loosening the analyzer here and not report an issue when there are comments on a separate line in the body. I don't think users would prefer expression body with comments in between the `=>` and the expression. @Evangelink can you file a Roslyn issue for this?

Reply:
Ticket created https://github.com/dotnet/roslyn/issues/50229